docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,33 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- TODO Rename this. We're not only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:<!-- TODO State here more clearly why this is useful as a feature -->
+
+|Permission|Organization Owner|Security Manager|
+|---|---|---|
+|Access Security and Risk Management|Yes|Yes|
+|Access originating items<!-- TODO reword all "originating items" instances for clarity --> (Codacy repositories, Jira issues)|Yes<!-- TODO admin may not have access here? Confirm -->|Inherits original permissions|
+|Manage integrations|Yes|<!-- TODO confirm if security managers cans manage integrations -->|
+|Assign and revoke the Security Manager role|Yes|No|
+
+To assign the Security Manager role to a user:
+
+1.  Access the **Security and risk management configuration** page.
```

Comment:
It's not clear how to this configuration page... 🤔

Reply:
👍 Updated with clearer instructions